_posts/2018-08-07-busconf-2018.md

```diff
@@ -0,0 +1,40 @@
+---
+author: Jorge Gueorguiev Garcia
+comments: true
+date: 2018-08-05 08:00:00 +00:00
+layout: post
+asset-type: post
+slug: busconf-2018
+title: Busconf 2018
+image:
+    src: /assets/custom/img/blog/bus-small.jpg
+tags:
+- conferences
+- functional programming
+---
+
+I love unconferences. The dynamics and the feeling is quite different from "normal" conferences. They tend to be quite unassuming in regards to the people or the talks. And I do love that some of the sessions are just people asking for someone to help them understand concepts or tools.
+
+I just attended [Busconf](http://www.bus-conf.org/index.html), which is a Functional Programming Unconference in its second year. And the experience has been greatly satisfying.
+
+### First, 
+the location, at the [SeminarZentrum](https://www.natuerlich-tagen.de/) in Rückersbach, a bit outside of Frankfurt, is an incredible and beautiful choice (if you want to have a view through google maps you can see it [here](https://www.google.com/maps/@50.0437687,9.1096571,3a,89.6y,323.65h,102.32t/data=!3m8!1e1!3m6!1sAF1QipPlVnBMK25MFA_-Sx1xjU050WdCOwQ-_bd69C4-!2e10!3e11!6shttps:%2F%2Flh5.googleusercontent.com%2Fp%2FAF1QipPlVnBMK25MFA_-Sx1xjU050WdCOwQ-_bd69C4-%3Dw203-h100-k-no-pi-0-ya167.02386-ro0-fo100!7i9728!8i4080)). I was even surprised that taxis would go over there. But to achieve a sense of community, is just perfect. There is nothing else to do but to mingle with the rest of the attendees. And I leave having met lots of interesting people.
```

Comment:
But to achieve a sense of community, *it* is just perfect.

Reply:
Done